config/env.ini

```diff
@@ -43,6 +44,8 @@ SPC_SKIP_PHP_VERSION_CHECK="no"
 SPC_SKIP_DOCTOR_CHECK_ITEMS=""
 ; extra modules that xcaddy will include in the FrankenPHP build
 SPC_CMD_VAR_FRANKENPHP_XCADDY_MODULES="--with github.com/dunglas/frankenphp/caddy --with github.com/dunglas/mercure/caddy --with github.com/dunglas/vulcain/caddy --with github.com/dunglas/caddy-cbrotli"
+; The display message for php version output (PHP >= 8.4 available)
+PHP_BUILD_PROVIDER="static-php-cli ${SPC_VERSION}"
```

Comment:
there are a couple more environment variables that can be set.

```
PHP_UNAME (probably doesn't make sense overwriting)
PHP_BUILD_SYSTEM
PHP_BUILD_PROVIDER
PHP_BUILD_COMPILER
PHP_BUILD_ARCH
```

And when EXTRA_LDFLAGS contains -release, the ./configure command is not displayed in `phpinfo();`

Reply:
Added PHP_BUILD_COMPILER.